resources/views/emails/messages/plain_text.blade.php

```diff
@@ -0,0 +1,6 @@
+Dear {!! isset($recipientName) ? $recipientName : $recipient->name !!},
+
+{{  strip_tags($body) }}
```

Comment:
I see what you mean, good point. Do the plain text emails still look normal after being processed like this?